index.ts

```diff
@@ -11,6 +11,14 @@ export type {PackVersionDefinition} from './types';
 export {PostSetupType} from './types';
 export type {SystemAuthentication} from './types';
 export type {WebBasicAuthentication} from './types';
+export type {CustomAuthentication} from './types';
+export type {CustomHeaderTokenAuthentication} from './types';
+export type {NoAuthentication} from './types';
+export type {HeaderBearerTokenAuthentication} from './types';
+export type {CodaApiBearerTokenAuthentication} from './types';
+export type {QueryParamTokenAuthentication} from './types';
+export type {MultiQueryParamTokenAuthentication} from './types';
+export type {AWSSignature4Authentication} from './types';
```

Comment:
Nit, please alphabetize.

Reply:
done and separated out into the authentication types themselves like we do with the others